substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/SecuritySubstitutions.java

```diff
@@ -317,6 +319,9 @@ static boolean isTrustedCryptoProvider(Provider provider) {
 @SuppressWarnings({"unused"})
 final class Target_javax_crypto_JceSecurity {
 
+    @Alias @TargetElement(onlyWith = JDK17_0_10OrLater.class)//
+    public static ReferenceQueue<Object> queue;
```

Comment:
Same comment as for the 23.0 PR with respect to the update version conditionals.

Reply:
Done